generators/resources/mappings.v1.json.template

```diff
@@ -3,6 +3,7 @@
     {{#each credentialKeys}}
     "{{../serviceKey}}_{{this}}" : {
         "searchPatterns": [
+            "user-provided:{{../serviceName}}_{{access ../map this ../nestedJSON}}",
```

Comment:
I think we need to use a `:` instead of `_` to separate name:credthing. That is how IBM Cloud Env (used by Node) expects the string to look.